main.py

```diff
@@ -3,5 +3,36 @@
 from django.db.models import QuerySet
 
 
+from db.models import Genre
+from db.models import Actor
+
+
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor.split()[0],
+            last_name=actor.split()[1]
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
If there is another guy with same surname but his name differs - his personal info should not be changed